File: php/examples/get_shopping_cart.php
<?php
require_once 'base_request.php';

// Example of GET /shopping-cart

$debug = true;
$company_id   = $_ENV['COMPANY_ID'];
$request_data = '';
$request_params = ['company_ids' => [$company_id]];

send_request('GET', '/shopping-cart', $request_params, $request_data);

File: csharp/examples/GetShoppingCart.cs
using System;
namespace Examples.examples
{
    // Example GET /shopping-cart
    
    public class GetShoppingCart : BaseRequest
    {
        private string _companyId;

        public GetShoppingCart()
		{
            dotenv.net.DotEnv.Load();
            _companyId = Environment.GetEnvironmentVariable("COMPANY_ID");
            Console.WriteLine($"GetShoppingCart: _companyId={_companyId}");
        }

        public GetShoppingCart(string companyId)
        {
            _companyId = companyId;
        }

        public override string SendRequest()
        {
            return GetRequest($"shopping-cart?company_ids={_companyId}");
        }
    }
}

